i18n/locales/en-GB/hourofcode/how-to/companies.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 怎样把《编程一小时》带到你的公司
+title: '怎样把《编程一小时》带到你的公司'
```

Comment:
This doesn't look right?

Reply:
https://hourofcode.com/us/gb/how-to/companies

Reply:
this is fine; just YAML being inconsistent

Reply:
Oh! Yeah, this is in English